ebpf-for-windows.sln

```diff
@@ -114,9 +114,9 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "unit_tests", "tests\unit\te
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "netsh_static", "libs\ebpfnetsh\ebpfnetsh.vcxproj", "{370E7D53-C97F-4077-A3D5-620C4FD87EED}"
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "test_ebpf_ext", "tests\ext\drv\test_ext.vcxproj", "{C8D46543-5AE5-4E66-B9CE-8B84588B1C9E}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "test_ebpf_ext", "tests\sample\ext\drv\test_ext.vcxproj", "{C8D46543-5AE5-4E66-B9CE-8B84588B1C9E}"
```

Comment:
can we rename it from "test_ebpf_ext" to "sample_ebpf_ext"?

Reply:
Yeah I would bulk find-replace every test_ext and TestExt to sample_ext and SampleExt after all other comments have been resolved.

Reply:
Renamed all test extension related binary, source and identifiers to "sample".